scripts/run-sam-task

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 set -euo pipefail
 
-COMPOSE_PROJECT_NAME=placeos docker-compose run --rm --no-deps init make sam "$@"
+COMPOSE_PROJECT_NAME=placeos docker-compose run --rm --no-deps init scripts/task "$@"
```

Comment:
Should this be `scripts/task`, or just `task` as per the [init readme](https://github.com/PlaceOS/init#example)?

Reply:
This is for the beta container (it's super old)

Reply:
it can be `task` now as the build artefacts are in the path now